public/views/partials/atoms/badge.liquid

```diff
@@ -0,0 +1,39 @@
+---
+metadata:
+  name: Badge
+  params:
+    tag: "div"
+    maxValue: 1000
+    value: 1000
+  styleguide:
+    - tag: "div"
+      maxValue: 20
+      value: 8
+    - tag: "div"
+      maxValue: 999
+      value: 1500
+    - tag: "div"
+      maxValue: 1999
+      value: 1500
+    - tag: "div"
+      value: 1500
+
+---
+{% liquid
+  assign tag = params.tag | default: 'div'
+  assign maxValue = params.maxValue | default: 0
+  assign value = params.value
+
+  if maxValue > 0 and maxValue < value
+    assign value = params.value | at_most: maxValue | append: "+"
+  endif
+
+  assign classes = 'px-1 py-1 leading-3 text-xs rounded-full min-w-[20] flex flex-col items-center bg-important text-inverted'
+%}
+
+
+<div class="inline-block">
+  <{{tag}} class="{{classes}}">
```

Comment:
if the main wrapper is always a <div> with `inline-block` then what is the use case for the `tag` param here?